src/dsp_tools/models/input_error.py

```diff
@@ -0,0 +1,134 @@
+from dataclasses import dataclass
+from typing import Protocol
+
+# pylint: disable=too-few-public-methods
+
+
+separator = "\n    "
+list_separator = "\n    - "
+
+
+class Problem(Protocol):
+    """Information about input errors."""
+
+    def execute_error_protocol(self) -> str:
```

Comment:
Currently that is all the method does. But in the future it may also create and save a df etc. (Since we adapt the type of information needed here). Still the method have to be called the same so that the protocol works. 

At first I create a private function called create error_msg which was called by this function. But since we do not (at least at this point) create dfs or other files, it seemed superfluous. I only wanted to create this option when needed and as this is a ontology file, there won't be that so many errors that it won't fit the console.